### PR TITLE
Add AOL support (Fixes #183)

### LINKED ISSUE #183: Adding AOL support
Adding enhancement to the project.

**Would you like to work on this?**
yes

### PR DESCRIPTION
**Pull request by bhaveshAn**
Fixes #183 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Added support for [AOL](https://www.aolsearch.com/) search engine
- 
screenshot from 2017-10-06 00-34-57

### CODE REVIEW

**Comment by bhaveshAn** [comment]
@cclauss Thanks !

File: app/scrapers/__init__.py [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
+from aol import AOL
 
 scrapers = {
     'g': Google(),
     'b': Bing(),
     'y': Yahoo(),
     'd': Duckduckgo(),
-    'a': Ask()
+    'a': Ask(),
+    'aol': AOL()
```
Comment (cclauss) [inline app/scrapers/__init__.py]: Same problem as #230

**Comment by bhaveshAn** [comment]
Please see [PR proof in YouTube](https://youtu.be/MRbAv6_1Kas)
@S2606 please review @niranjan94 please merge :)

**Comment by mariobehling** [comment]
Please refer to https://github.com/fossasia/query-server/issues/183#issuecomment-335227518

**Pull request closed without merging**